presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/Memo.java

```diff
@@ -69,12 +72,18 @@
     private final int rootGroup;
 
     private final Map<Integer, Group> groups = new HashMap<>();
-
+    private final Optional<LogicalPropertiesProvider> logicalPropertiesProvider;
     private int nextGroupId = ROOT_GROUP_REF + 1;
```

Comment:
nits: keep the empty line between final and non final variables.

Reply:
Done.